### ISSUE TITLE
Comments for @hkaman7 's notebook

### ISSUE THREAD

**Issue by kteavery** [alburke/ams-2020-ml-python-course#5]
@hkaman7 I realize you might not be completely done with your notebook, but I wanted to give some feedback and ideas while we have time to look at it and adjust things. 

* I think you're lecture 3, not lecture 2.
* Add a references section
*  I crashed on the `import utils` line. This isn't really an issue; you can just have everyone add to the PYTHONPATH. I personally like adding a little boilerplate at the top to avoid changing environment variables manually: 
```
import sys
from os.path import dirname, abspath
sys.path.insert(1, abspath('')+"/introduction_to_machine_learning")
...
import Lecture_2.utils as utils
```
* In the "Read Data" section, I already explore the contents of one of the files in my lecture. 
* Amanda talks about normalization in her section, and I touch upon it briefly in relation to content in my section. Review what's in those sections and adjust content from your lecture as necessary.
* I go over linear regression, hyperparameter search, and L1 and L2 regularization in my section. I don't go over ElasticNet.
* At the moment, I introduce the idea of binarization, ROC curves, attribute diagrams, and performance diagrams at the end of my presentation (as we discussed earlier). Feel free to look at my stuff to see how you want to transition. I can delete/alter content at the end of my lecture depending on what you need for yours. 
* Perhaps you could briefly introduce artificial neural nets to set up for Karthik's deep learning talk? (Very basic ANNs aren't necessarily deep learning, so I feel like that would go in your section fine.)
* I would explain SVMs as a concept, even if you don't have time to do a full-fledged experiment with one.

**Comment by hkaman7** [comment]
Hi Kate!

Thanks for the feedback, that’s very helpful l.

Yes, it’s not completely done yet!
Ok, I will check everything and get back to you.

Thanks
Hamid

**Comment by hkaman7** [comment]
Hi Kate!

I review both our presentations!

1. I will change the name as lecture3, you are right!
2. The references are added, just need to update the notebook.
3. Thanks for code! helped!
4. I will skip the section of normalization and data preparation, and to be
the same I will use the same code provided by David for scaling part as you
did. Just try to introduce these functions in sklearn.
5. You already cover the lasso and the ridge penalty, I will go over the
ElasticNet only for logistic regression.
6. Good! you covered the evaluation metrics for classification, the only
thing is a learning curve which I will take care of it.
7. I will explain the SVM and NN, everything sounds good!

Thanks for your help
Hamid
